litmus/director/TCID-DIR-INSTALL-ON-LOCAL-HP/README.md

```diff
@@ -0,0 +1,47 @@
+#  Deploying Director on-prem
+<b>tcid:</b> TCID-DIR-INSTALL-ON-LOCAL-HP <br>
+<b>name:</b>  Deploy director on-prem <br>
+
+## Details
+
+- Install DOP using helm with SSD underheath
+
+## Prerequisites
+
+- K8s should be installed in the cluster.
+- Every component of DOP cluster should be healthy and running.
```

Comment:
This is not a pre-requisite. This should be the result.

Reply:
One more pre-requisite is there - 
- Openebs should be installed on the cluster.